handler/core/src/main/java/io/knotx/fragments/task/factory/node/action/ActionNodeFactory.java

```diff
@@ -88,7 +92,13 @@ public String getId() {
 
       @Override
       public void apply(FragmentContext fragmentContext, Handler<AsyncResult<FragmentResult>> handler) {
-        action.apply(fragmentContext, handler);
+        metadata.markProcessingStart();
```

Comment:
This is hacky. Metadata is filled when node is processed - how about getting this information from EventLog filled by TaskEngine? Would require a few adjustments there.

Reply:
I know it's hacky, I'm not sure what would be the best way. It should be discussed.
Btw, is there such information in the event log? I must have missed it

Reply:
Currently `EventLogEntry` contains information about the finishing timestamp of any given node. I think it should be possible to enhance this information with start timestamp with a reasonable effort.
However, traversing through the subgraphs to find start/end time for the containing composite node would be more difficult. Perhaps this is not required at all?

Reply:
I think that @marcinus is right. I think that we should keep the information about node processing start / stop in the fragment event log.

@Voycawojka please check my proposal solution: `feature/node-processing-times`.